Build/Build.cs

```diff
@@ -40,14 +41,18 @@
 
 // ReSharper disable RedundantTypeArgumentsOfMethod
 
-class Build : RemotionBuild, IDependDB
+class Build : RemotionBuild, IDependDB, ITest
 {
   [Parameter(ValueProviderMember = nameof(SupportedTestBrowsers), Separator = "+")]
   public string[] TestBrowsers { get; set; } = [];
 
   [Parameter(ValueProviderMember = nameof(SupportedTestSqlServers), Separator = "+")]
   public string[] TestSqlServers { get; set; } = [];
 
+  [CanBeNull] private TestMatrix _databaseTestMatrix;
+  [CanBeNull] private TestMatrix _normalTestMatrix;
+  [CanBeNull] private TestMatrix _webTestMatrix;
```

Comment:
we named this `WebTestingTest...` in other places (see below). We should keep the naming consistent. If the short version is preferred, fix that one, too.